### PR TITLE
Fixing the issue on packager (failing with siblings error)

### PR DESCRIPTION
**Pull request by shiranGinige**
The packager was failing with below error after a fresh clone

npm ERR! peerinvalid The package react@15.1.0 does not satisfy its siblings' peerDependencies requirements!

Changed the dependancy for **^react@15.1.0-alpha.1** to **^react@15.1.0** and it was fixed.

### CODE REVIEW

**Comment by facebook-github-bot-3** [comment]
By analyzing the blame information on this pull request, we identified @cpojer and @davidaurelio to be potential reviewers.

**Comment by facebook-github-bot-3** [comment]
Thank you for your pull request and welcome to our community. We require contributors to sign our Contributor License Agreement, and we don't seem to have you on file. In order for us to review and merge your code, please sign up at https://code.facebook.com/cla - and if you have received this in error or have any questions, please drop us a line at cla@fb.com. Thanks!

**Comment by facebook-github-bot-0** [comment]
@shiranGinige updated the pull request.

**Comment by facebook-github-bot-4** [comment]
Thank you for signing our Contributor License Agreement. We can now accept your code for this (and any) Facebook open source project. Thanks!

File: package.json [hunk 1]
```diff
@@ -191,7 +191,7 @@
     "flow-bin": "^0.25.0",
     "jest": "12.1.1",
     "portfinder": "0.4.0",
-    "react": "^15.1.0-alpha.1",
+    "react": "^15.1.0",
```
Comment (bestander) [inline package.json]: let's make it strict so that it does not go out of sync again

**Comment by bestander** [comment]
Good point, let's go to non alpha now that it is released

File: package.json (hunk 1 above)
Comment (shiranGinige) [inline package.json]: done.

**Comment by bestander** [comment]
Thanks, the Travis build error is unrelated.
Importing

**Comment by bestander** [comment]
@facebook-github-bot shipit

**Comment by facebook-github-bot-0** [comment]
@shiranGinige updated the pull request.

**Pull request closed without merging**